vendor/ocaml-4.12/dune

```
@@ -0,0 +1,44 @@
+(library
+ (name ocaml_412)
+ (flags
+  (:standard -w -9 -open Parser_shims))
+ (libraries compiler-libs.common menhirLib parser_shims))
```

```suggestion
 (libraries menhirLib parser_shims))
```
Covered by `parser-shims`